pyproject.toml

```diff
@@ -32,18 +35,26 @@ all = ["boto3", "pyyaml"]
 pytest = "*"
 pytest-mock = "*"
 pytest-cov = "^2.8.1"
-mypy = "^0.770"
-taskipy = "^1.2.1"
-safety = "^1.9.0"
-black = "^19.10b0"
-isort = "^4.3.21"
+mypy = "==0.*"
+safety = "*"
+black = {version = ">=19.10b0", allow-prereleases = true}
+taskipy = "^1.4.0"
+isort = "^5.6.4"
 mkdocs = "^1.1"
 mkdocs-material = "^5.1.5"
 markdown-include = "^0.5.1"
 mkdocstrings = "^0.11.0"
 
 [tool.taskipy.tasks]
-check = "isort --recursive --apply && black . && safety check && mypy flex_config && pytest --cov flex_config"
+check = """
+    autoflake -i -r --expand-star-imports --remove-all-unused-imports --remove-unused-variables flex_config\
```

Comment:
<3

A PR in openapi-python-client revealed to me the `--ignore-init-module-imports` flag which does exactly what it says and _I think_ resolves the issue we had in FRT schemas imports. Might want to make it part of our standard stuff.